conda/recipes/libwholegraph/meta.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2019-2023, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set cuda_version = '.'.join(environ['RAPIDS_CUDA_VERSION'].split('.')[:2]) %}
+{% set cuda_major = cuda_version.split('.')[0] %}
+{% set cuda_spec = ">=" + cuda_major ~ ",<" + (cuda_major | int + 1) ~ ".0a0" %} # i.e. >=11,<12.0a0
+{% set date_string = environ['RAPIDS_DATE_STRING'] %}
+
+package:
+  name: libwholegraph-split
+
+source:
+  git_url: ../../..
+
+build:
+  script_env:
+    - AWS_ACCESS_KEY_ID
+    - AWS_SECRET_ACCESS_KEY
+    - AWS_SESSION_TOKEN
+    - BUILD_ABI
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_GENERATOR
+    - PARALLEL_LEVEL
+    - SCCACHE_BUCKET
+    - SCCACHE_IDLE_TIMEOUT
+    - SCCACHE_REGION
+    - SCCACHE_S3_KEY_PREFIX=libwholegraph-aarch64 # [aarch64]
+    - SCCACHE_S3_KEY_PREFIX=libwholegraph-linux64 # [linux64]
+    - SCCACHE_S3_USE_SSL
+    - SCCACHE_S3_NO_CREDENTIALS
+
+# Here we duplicate the dependencies so conda will populate both _build_env
+# and _h_env_placeholder directories. This is needed to run clang-tidy.
+# We need both build and host, or conda will only use _h_env_placeholder
+# directory, which is removed after a build. Having only _build_env is not
+# enough as CMake will not search for dependencies in this directory.
+# This needs to be removed once CI support from sources builds
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }} {{ cuda_version }}
+    - {{ compiler('cxx') }}
+    - cmake {{ cmake_version }}
+    - ninja
+    - sysroot_{{ target_platform }} {{ sysroot_version }}
+  host:
+    - cmake {{ cmake_version }}
+    - cuda-nvtx ={{ cuda_version }}
+    - cudatoolkit ={{ cuda_version }}
+    - doxygen {{ doxygen_version }}
+    - gmock {{ gtest_version }}
+    - gtest {{ gtest_version }}
+    - libcugraphops ={{ minor_version }}
```

Comment:
Does `libwholegraph` depend on `libcugraphops`?

Reply:
No, `libwholegraph` doesn't depend on `libcugraphops`, I think we can just remove this line. I will verify that and submit a PR for the `libcugraphops` dependency removed yaml file.

Reply:
> Does `libwholegraph` depend on `libcugraphops`?

Updated in PR #41 , also added CUDA 12